docs/pages/versions/unversioned/sdk/media-library.md

```diff
@@ -229,7 +239,8 @@ Subscribes for updates in user's media library.
 
 #### Arguments
 
-- **listener (_function_)** -- A callback that is called when any assets have been inserted or deleted from the library. **On Android** it's invoked with an empty object. **On iOS** it's invoked with an object that contains following keys:
+- **listener (_function_)** -- A callback that is fired when any assets have been inserted or deleted from the library, or when the user changes which assets they're allowing access to. **On Android** it's invoked with an empty object. **On iOS** it's invoked with an object containing following keys:
+  - **assetPermissionsChanged (_boolean_)** -- Flag indicating a permissions change occurred in the user's limited library selection. If `true`, the user has changed access to individual assets in their library, and you should reload all used assets to comply with the new permissions.
```

Comment:
```suggestion
  - **assetsPermissionsChanged (_boolean_)** -- Flag indicating a permissions change occurred in the user's limited library selection. If `true`, the user has changed access to individual assets in their library, and you should reload all used assets to comply with the new permissions.
```
Yeah, I like this name more then the previous one 😉 
I suggest going with the plural form as there's number of assets rather than single one 🤔